.github/workflows/pytest.yml

```
@@ -21,3 +21,27 @@
         sudo mkdir /var/log/s3_upload
         sudo chmod o+w /var/log/s3_upload
         pytest -v --cov --count 10 --random-order tests/unit/
+    - name: Get test coverage %
+      run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:4:26: Double quote to prevent globbing and word splitting

(shellcheck)